homeassistant/components/geniushub/sensor.py

```diff
@@ -38,6 +39,8 @@
 class GeniusDevice(Entity):
     """Representation of a Genius Hub sensor."""
 
+    parallel_updates = True
```

Comment:
This attribute should preferably be set indirectly by setting the `PARALLEL_UPDATES` module level constant in the platform.

By default entities that have an `async_update` method will update in parallel.

https://github.com/home-assistant/home-assistant/blob/1e149a704bb38866828920e186190d7d2e7d0586/homeassistant/helpers/entity_platform.py#L242-L256

Reply:
Ah! Then I shall remove it.